### PR TITLE
test: Port read_window_aggregate query_tests to end-to-end tests

### PR DESCRIPTION
**Pull request by carols10cents**
Connects to https://github.com/influxdata/influxdb_iox/issues/6617.

### CODE REVIEW

File: influxdb_iox/tests/end_to_end_cases/querier/influxrpc/read_window_aggregate.rs [hunk 1]
```diff
@@ -121,3 +123,262 @@ async fn do_read_window_aggregate_test(
     .run()
     .await
 }
+
+#[tokio::test]
```
Comment (carols10cents) [inline influxdb_iox/tests/end_to_end_cases/querier/influxrpc/read_window_aggregate.rs:127]: Was `test_read_window_aggregate_nanoseconds` https://github.com/influxdata/influxdb_iox/blob/5b14caa780b77c6607c84ad38578ed4d28f15e86/query_tests/src/influxrpc/read_window_aggregate.rs#L53-L79

File: influxdb_iox/tests/end_to_end_cases/querier/influxrpc/read_window_aggregate.rs [hunk 2]
```diff
@@ -121,3 +123,262 @@ async fn do_read_window_aggregate_test(
...
+            "FloatPointsFrame, timestamps: [200, 400, 600], values: \"70,71.5,73\"",
+            "SeriesFrame, tags: _field=temp,_measurement=h2o,city=LA,state=CA, type: 0",
+            "FloatPointsFrame, timestamps: [200, 400, 600], values: \"90,91.5,93\"",
+        ],
+    })
+    .run()
+    .await;
+}
+
+#[tokio::test]
```
Comment (carols10cents) [inline influxdb_iox/tests/end_to_end_cases/querier/influxrpc/read_window_aggregate.rs:149]: Was `test_read_window_aggregate_nanoseconds_measurement_pred` https://github.com/influxdata/influxdb_iox/blob/5b14caa780b77c6607c84ad38578ed4d28f15e86/query_tests/src/influxrpc/read_window_aggregate.rs#L82-L111

File: influxdb_iox/tests/end_to_end_cases/querier/influxrpc/read_window_aggregate.rs [hunk 3]
```diff
@@ -121,3 +123,262 @@ async fn do_read_window_aggregate_test(
...
```
Comment (carols10cents) [inline influxdb_iox/tests/end_to_end_cases/querier/influxrpc/read_window_aggregate.rs:172]: Was `test_read_window_aggregate_nanoseconds_measurement_count` https://github.com/influxdata/influxdb_iox/blob/5b14caa780b77c6607c84ad38578ed4d28f15e86/query_tests/src/influxrpc/read_window_aggregate.rs#L114-L138

File: influxdb_iox/tests/end_to_end_cases/querier/influxrpc/read_window_aggregate.rs [hunk 4]
```diff
@@ -121,3 +123,262 @@ async fn do_read_window_aggregate_test(
...
+            "SeriesFrame, tags: _field=temp,_measurement=h2o,city=Cambridge,state=MA, type: 1",
+            "IntegerPointsFrame, timestamps: [200, 400, 600], values: \"1,2,1\"",
+            "SeriesFrame, tags: _field=temp,_measurement=h2o,city=LA,state=CA, type: 1",
+            "IntegerPointsFrame, timestamps: [200, 400, 600], values: \"1,2,1\"",
+        ],
+    })
+    .run()
+    .await;
+}
+
```
Comment (carols10cents) [inline influxdb_iox/tests/end_to_end_cases/querier/influxrpc/read_window_aggregate.rs:193]: Was `test_grouped_series_set_plan_group_aggregate_min_defect_2697` https://github.com/influxdata/influxdb_iox/blob/5b14caa780b77c6607c84ad38578ed4d28f15e86/query_tests/src/influxrpc/read_window_aggregate.rs#L140-L171

File: influxdb_iox/tests/end_to_end_cases/querier/influxrpc/read_window_aggregate.rs [hunk 5]
```diff
@@ -121,3 +123,262 @@ async fn do_read_window_aggregate_test(
...
+            timestamps: [1609459201000000009, 1609459201000000015, 1609459201000000022], \
+            values: \"4,6,1.2\"",
+            "SeriesFrame, tags: _field=foo,_measurement=mm,section=2b, type: 0",
+            "FloatPointsFrame, timestamps: [1609459201000000002], values: \"2\"",
+        ],
+    })
+    .run()
+    .await;
+}
+
```
Comment (carols10cents) [inline influxdb_iox/tests/end_to_end_cases/querier/influxrpc/read_window_aggregate.rs:225]: Was `test_grouped_series_set_plan_group_aggregate_sum_defect_2697` https://github.com/influxdata/influxdb_iox/blob/5b14caa780b77c6607c84ad38578ed4d28f15e86/query_tests/src/influxrpc/read_window_aggregate.rs#L173-L203

File: influxdb_iox/tests/end_to_end_cases/querier/influxrpc/read_window_aggregate.rs [hunk 6]
```diff
@@ -121,3 +123,262 @@ async fn do_read_window_aggregate_test(
...
+            timestamps: [1609459201000000010, 1609459201000000020, 1609459201000000030], \
+            values: \"4,6,1.2\"",
+            "SeriesFrame, tags: _field=foo,_measurement=mm,section=2b, type: 0",
+            "FloatPointsFrame, timestamps: [1609459201000000010], values: \"2\"",
+        ],
+    })
+    .run()
+    .await;
+}
+
```
Comment (carols10cents) [inline influxdb_iox/tests/end_to_end_cases/querier/influxrpc/read_window_aggregate.rs:256]: Was `test_grouped_series_set_plan_group_aggregate_filter_on_field` https://github.com/influxdata/influxdb_iox/blob/5b14caa780b77c6607c84ad38578ed4d28f15e86/query_tests/src/influxrpc/read_window_aggregate.rs#L205-L236

File: influxdb_iox/tests/end_to_end_cases/querier/influxrpc/read_window_aggregate.rs [hunk 7]
```diff
@@ -121,3 +123,262 @@ async fn do_read_window_aggregate_test(
...
+            values: \"4,11.24\"",
+            "SeriesFrame, tags: _field=foo,_measurement=mm,section=2b, type: 0",
+            "FloatPointsFrame, timestamps: [1609459201000000010], values: \"2\"",
+        ],
+    })
+    .run()
+    .await;
+}
+
+#[tokio::test]
```
Comment (carols10cents) [inline influxdb_iox/tests/end_to_end_cases/querier/influxrpc/read_window_aggregate.rs:285]: Was `test_read_window_aggregate_overflow` https://github.com/influxdata/influxdb_iox/blob/5b14caa780b77c6607c84ad38578ed4d28f15e86/query_tests/src/influxrpc/read_window_aggregate.rs#L238-L261

File: influxdb_iox/tests/end_to_end_cases/querier/influxrpc/read_window_aggregate.rs [hunk 8]
```diff
@@ -121,3 +123,262 @@ async fn do_read_window_aggregate_test(
...
+            "FloatPointsFrame, timestamps: [1609459201000000015], values: \"6\"",
+            "SeriesFrame, tags: _field=foo,_measurement=mm, type: 0",
+            "FloatPointsFrame, timestamps: [1609459201000000005], values: \"3\"",
+        ],
+    })
+    .run()
+    .await;
+}
+
+#[tokio::test]
```
Comment (carols10cents) [inline influxdb_iox/tests/end_to_end_cases/querier/influxrpc/read_window_aggregate.rs:308]: Was `test_read_window_aggregate_with_periods` https://github.com/influxdata/influxdb_iox/blob/5b14caa780b77c6607c84ad38578ed4d28f15e86/query_tests/src/influxrpc/read_window_aggregate.rs#L264-L287

**Review by carols10cents — COMMENTED** [review]
Matching up tests

**Review by alamb — APPROVED** [review]
I reviewed the two sets of cases carefully and agree @carols10cents  has done a great job of faithfully porting them over. Thanks!

**Pull request merged**